sys/usb/usbus/msc/msc.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (C) 2019-2021 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup usbus_msc
+ * @{
+ *
+ * @author  Dylan Laduranty <dylan.laduranty@mesotic.com>
+ * @}
+ */
+
+#include "usb/descriptor.h"
+#include "usb/usbus.h"
+#include "usb/usbus/control.h"
+
+#include "usb/msc.h"
+#include "usb/usbus/msc.h"
+#include "usb/usbus/msc/scsi.h"
+#include "board.h"
+
+#include <string.h>
+
+#define ENABLE_DEBUG    0
+#include "debug.h"
+
+#include "mtd.h"
+extern mtd_dev_t *mtd0;
```

Comment:
isn't that already defined by `board.h`?

How about

```C
#ifndef MTD_MSC
#define MTD_MSC MTD_0
#endif
```

and then use that instead of `mtd0`.

Reply:
Done.